syscalls.go

```diff
@@ -277,7 +277,19 @@ func bpfMapGetNextKey(m *internal.FD, key, nextKeyOut internal.Pointer) error {
 		value: nextKeyOut,
 	}
 	_, err = internal.BPF(_MapGetNextKey, unsafe.Pointer(&attr), unsafe.Sizeof(attr))
-	return err
+	return wrapMapError(err)
+}
+
+func wrapMapError(err error) error {
+	if err == nil {
+		return nil
+	}
+
+	if err == unix.ENOENT {
```

Comment:
maybe, we can use `xerrors.Is(err, unix.ENOENT)` here?

Reply:
Good point!